apps/sms/services/js/conversation/conversation_client.js

```diff
@@ -0,0 +1,100 @@
+/* global bridge,
+          performance,
+          streamClient,
+          Thread
+*/
+
+(function(exports) {
+  'use strict';
+
+  const debug = 0 ?
+    (arg1, ...args) => console.log(`[ConversationClient] ${arg1}`, ...args):
+    () => {};
+
+  const mark = 0 ?
+    (markName) => performance.mark(`[ConversationClient] ${markName}`):
```

Comment:
maybe use `window.performance.mark` so that you don't need to add `performance` in the global line above